test/tests/functional/pbs_test_tpp.py

```diff
@@ -171,31 +168,30 @@ def pbs_restart(self, host_name):
         pi = PBSInitServices(hostname=host_name)
         pi.restart()
 
-    def set_conf(self, host_name, attribs):
+    def set_conf(self, host_name, conf_param):
         """
         This function sets attributes in pbs.conf file
         :param host_name: Name of the host on which pbs.conf
                           has to be updated
         :type host_name: String
-        :param attribs: Attributes to be updated in pbs.conf
-        :type attribs: Dictionary
+        :param conf_param: Attributes to be updated in pbs.conf
+        :type conf_param: Dictionary
         """
         pbsconfpath = self.du.get_pbs_conf_file(hostname=host_name)
         self.du.set_pbs_config(hostname=host_name, fin=pbsconfpath,
-                               confs=attribs)
+                               confs=conf_param)
         self.pbs_restart(host_name)
 
-    def unset_pbs_conf(self, host_name):
+    def unset_pbs_conf(self, host_name, conf_param):
         """
         To functions unsets "PBS_LEAF_ROUTERS" from pbs.conf file
         """
```

Comment:
the docstring is not correct as we have made changes now.
adding details like we did for set_conf() would also help.
for consistency - shall we name these functions set_conf, unset_conf OR set_pbs_conf, unset_pbs_conf?